packages/bpk-mixins/src/mixins/_chips.scss

```diff
@@ -24,143 +24,61 @@ $bpk-spacing-v2: true;
   padding: 0 bpk-spacing-base();
   align-items: center;
   border: none;
+  border-radius: $bpk-button-border-radius;
 
-  @include bpk-border-radius-pill;
+  &--on-dark {
+    background-color: $bpk-core-primary-day;
+    color: $bpk-text-on-dark-day;
 
-  &--light {
-    background-color: $bpk-color-white;
-    color: $bpk-color-sky-gray;
-
-    @include bpk-border-sm($bpk-color-sky-gray-tint-03);
+    @include bpk-border-sm($bpk-line-on-dark-day);
 
     @include bpk-hover {
-      background-color: $bpk-color-sky-gray-tint-07;
-    }
-
-    &:active:not(:disabled) {
```

Comment:
any reason this needs to be removed? 🤔 `active` style is different looking at Figma so we probably still need this?

Reply:
Yes. The latest figma is not designed to have an active state. Removing acitve state does not affect the Chip component with disabled state. 😄